autoconnect/autoconnect-common/src/protocol.rs

```diff
@@ -123,6 +127,10 @@ pub enum ServerMessage {
 
     Notification(Notification),
 
```

Comment:
Introducing new protocol messages should be kept behind a feature flag until we can properly verify that there are no problems with current or legacy UserAgents.